pkg/k8s/apps/translate.go

```diff
@@ -698,12 +698,20 @@ func TranslateOktetoDevSecret(spec *apiv1.PodSpec, secret string, secrets []mode
 			},
 		},
 	}
+	idx := 0
 	for i, s := range secrets {
+		key := s.GetKeyName()
+		path := s.GetFileName()
+		if strings.Contains(key, ".stignore") {
+			idx++
+			key = fmt.Sprintf("%s-%d", key, idx)
+			path = fmt.Sprintf("%s-%d", path, idx)
```

Comment:
shouldn't the path be based on the destination mount path?

Reply:
We create a new file based on the one provided by the user here:
https://github.com/okteto/okteto/blob/4bba2288cc517499bf7788357520a7ff585990d0/cmd/up/stignore.go#L50

Reply:
That path is getting the name of the secret, which was the same for all the `.stignore` files and caused the error